configure.py

```diff
@@ -2393,8 +2410,7 @@ def release_date(datestamp):
 if __name__ == '__main__':
     try:
         main()
-    except Exception as e:
-        import traceback
+    except ConfigureError as e:
```

Comment:
This last outmost `except`  should stay `Exception` - we want to capture all runtime errors here, not just ones we generated.